scheduler/src/cook/mesos/api.clj

```diff
@@ -1968,6 +1969,7 @@
     (instance? ServerSocket v) (str v)
     (instance? Var v) (str v)
     (instance? RiemannReporter v) (str v)
+    (instance? ConsoleReporter v) (str v)
```

Comment:
Why is this needed? If `ConsoleReporter` is needed, then should we make a single clause covering `com.codahale.metrics.ScheduledReporter` and remove the clauses for `ConsoleReporter ` and `RiemannReporter `?

Reply:
This is needed because otherwise calls to `/settings` fail when the console reporter is turned on.

Combined to use `ScheduledReporter`.